L/libisal/build_tarballs.jl

```diff
@@ -0,0 +1,46 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libisal"
```

Comment:
https://docs.binarybuilder.org/stable/building/#Name

> This is the name that will be used in the tarballs and for the JLL package. It should be the name of the upstream package, not for example that of a specific library or executable provided by it, even though they may coincide.

If you're going to rename this variable, remember that the name of the directory must match exactly the `name`.

Reply:
I used isa_l now instead of the actual name isa-l.